gradle.properties

```diff
@@ -0,0 +1,4 @@
+sprintBootPluginVersion=2.7.3
```

Comment:
we can also store these versions inside the `build.gradle` itself like [here](https://github.com/dimagi/commcare-android/blob/master/build.gradle#L32). I like that style better since versions are maintained in the same file and also remove dependency on `gradle.propeties` which can be modified locally to add more attributes sometimes. Thoughts ?

Reply:
I'm open to changing this but as far as I know that doesn't work for plugin versions. From the docs: `«plugin version» must be constant, literal, strings`.

This is the recommended way to manage plugin versions: https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_version_management

Local gradle properties can still be provided via the `gradle.properties` file in the users home directory.

Reply:
Makes sense, happy to stick with current style in that case. Thanks!